src/h5web/breadcrumbs/Crumb.tsx

```diff
@@ -0,0 +1,34 @@
+import type { ReactElement } from 'react';
+import { FiChevronRight } from 'react-icons/fi';
+import styles from './BreadcrumbsBar.module.css';
+
+interface Props {
+  name: string;
+  onClick?: () => void;
+  isCurrent?: boolean;
+}
+
+function Crumb(props: Props): ReactElement {
+  const { name, onClick, isCurrent } = props;
+
+  if (!onClick) {
```

Comment:
We've got a case of mutually exclusive props: `onClick` and `isCurrent`. Since `onClick` is needed, I would just compute `const isCurrent = !onClick`. Unless you think of a better solution.

Reply:
Alternatively, you could keep the last crumb inlined in the parent.

Reply:
`const isCurrent = !onClick` is fine. 

While it is a big assumption to tell that the crumb is the current one if there is no `onClick` behaviour, we can easily revert in the future.

Reply:
Nah, it is clearer to keep it inlined in fact.